### ISSUE TITLE
solana-verify verify-from-repo with --remote fails

### ISSUE THREAD

**Issue by SushantChandla** [Ellipsis-Labs/solana-verifiable-build#108]
While running:`solana-verify verify-from-repo --library-name gamma --program-id GAMMA7meSFWaBXF25oSUgmGRwaW6sCMFLmBNiMSdbHVT https://github.com/GooseFX1/gamma-swap --commit-hash 9a636372886ccad94d4bd524439bba63ca3a942e  --remote`
I get:
```rust
Sending verify command to remote machine...
Encountered an error while attempting to send the job to remote
Error: "{\"status\":\"error\",\"error\":\"The PDA associated with the given signer was not found. Please try again with a valid signer.\"}"
```
I tried various versions. This is only a problem with ` --remote`.
The transaction to add PDA without --remote is here: https://explorer.solana.com/tx/468oQtpTLJqbrHvxfW54UVFkCwbx4tE6yN23ffD2rQAQwWahvEFdS41iL67Df7b5TpaYCPgHN7f7tfvVYNLFrV9c

**Comment by ngundotra** [comment]
Looking into this. Thanks for flagging this

**Comment by ngundotra** [comment]
Good news, your program is verified via that PDA.

Less great news, in order to have the PDA be recognized as "canonical" verification, it has to be uploaded by the program authority. We are exposing a fix for this in soon to be released `v0.4.0`.

You can use the same arguments in `solana-verify export-pda-tx --compute-unit-price 0 --encoding base58` to get a base58 encoded transaction that you can propose to your Squads multisig.

**Comment by SushantChandla** [comment]
Alright, Thanks!
